app/src/kiwix/play/contact-website.txt

```diff
@@ -1 +0,0 @@
-http://www.kiwix.org
```

Comment:
Why were these files added and are now being removed?

Reply:
This was a part of an initiative of storing the play store listing in source control, it is a feature of Gradle Play Publisher, you can run the `bootstrap` command and it downloads all the assets but it has been more of a pain than anything and it seems like we should just manage it on the play store itself. 
Also they were in the wrong folder. The `kiwix` flavour/sourceSet no longer exists post modularisation so they weren't being used.

I'll admit to being fairly scatterbrained post the custom apps work and it should have been its own pr.